global.json

```diff
@@ -0,0 +1,14 @@
+{
+  "tools": {
+    "dotnet": "3.0.100",
+    "runtimes": {
+      "dotnet": [
+        "2.1.11"
```

Comment:
Do we actually need this or did you copy it from a different repo? This is only sometimes needed depending on the frameworks targeted by the projects, so it may not be needed in your case.

Reply:
Testing with the runtime not specified.